unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Utils/CoordinateUtils.cs

```diff
@@ -2,7 +2,7 @@
 
 public class CoordinateUtils
 {
-    private const string COORDINATE_MATCH_REGEX = @"(?<!\S)[-]?\d{1,3},[-]?\d{1,3}(?!\S)";
+    private const string COORDINATE_MATCH_REGEX = @"(?<!\w)(?<!<\/noparse>)[-]?\d{1,3},[-]?\d{1,3}(?!\w)(?!<\/\/noparse>)";
```

Comment:
no need to escape special characters like `\/` since the string is marked with `@`